xml/runtime_config.xml

```diff
@@ -10,60 +10,259 @@
-    cluster.
+    You can update the value of configuration options with the following
+    command:
    </para>
-  </warning>
+<screen>ceph config set <replaceable>DAEMON</replaceable> <replaceable>OPTION</replaceable> <replaceable>VALUE</replaceable></screen>
+   <para>
+    For example, to adjust the debugging log level on a specific OSD, run:
+   </para>
+<screen>&prompt.cephuser;ceph config set osd.123 debug_ms 20</screen>
+   <note>
+    <para>
+     If the same option is also customized in a local configuration file, the
+     monitor setting will be ignored because it has lower priority than the
+     configuration file.
+    </para>
+   </note>
+   <sect3 xml:id="cha-ceph-configuration-db-runtime-override">
+    <title>Overriding Values</title>
+    <para>
+     You can temporarily modify an option value using the
+     <command>tell</command> or <command>daemon</command> subcommands. Such
+     modification only affect the running process and is discarded after the
+     daemon or process restarts.
+    </para>
+    <para>
+     There are two ways to override values:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       Use the <command>tell</command> subcommand to send a message to a
+       specific daemon from any cluster node:
+      </para>
+<screen>ceph tell <replaceable>DAEMON</replaceable> config set <replaceable>OPTION</replaceable> <replaceable>VALUE</replaceable></screen>
+      <para>
+       For example:
+      </para>
+<screen>&prompt.cephuser;ceph tell osd.123 config set debug_osd 20</screen>
+      <tip>
+       <para>
+        The <command>tell</command> subcommand accepts wildcards as daemon
+        identifiers. For example, to adjust the debug level on all OSD daemons,
+        run:
+       </para>
+<screen>&prompt.cephuser;ceph tell osd.* config set debug_osd 20</screen>
+      </tip>
+     </listitem>
+     <listitem>
+      <para>
+       Use the <command>daemon</command> subcommand to connect to a specific
+       daemon process via a socket in <filename>/var/run/ceph</filename> from
+       the node where the process is running:
+      </para>
+<screen>ceph daemon <replaceable>DAEMON</replaceable> config set <replaceable>OPTION</replaceable> <replaceable>VALUE</replaceable></screen>
+      <para>
+       For example:
+      </para>
+<screen>&prompt.cephuser;ceph daemon osd.4 config set debug_osd 20</screen>
+     </listitem>
+    </itemizedlist>
+    <tip>
+     <para>
+      When viewing runtime settings with the <command>ceph config
+      show</command> command (see
+      <xref linkend="cha-ceph-configuration-db-runtime-view"/>), temporarily
+      overridden values will be shown with a source
+      <literal>override</literal>.
+     </para>
+    </tip>
+   </sect3>
+   <sect3 xml:id="cha-ceph-configuration-db-runtime-view">
+    <title>Viewing Runtime Settings</title>
+    <para>
+     View all options set for a daemon:
```

Comment:
How would you feel about "To view..." instead of just "View" in each of these?

"View" sounds like an order to me, whereas "To view" sounds like a suggestion.